andela/isildur-rc
# #151207885 Digital Products

#### What does this PR do?
Adds Digital products to the platform

#### Description of Task to be completed?
Admin/Vendors can sell digital products on the platform. Clients/guests can download these digital products after payment.

#### How should this be manually tested?
Admin should add a digital product, by selecting the product type of digital product and client should buy the product and download it after payment

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/151207885
